RdlMigration/ConvertRDL.cs

```diff
@@ -560,6 +577,50 @@ private XElement CreateDataSourceNode(XDocument doc, DataSourceDefinition curren
             return currNode;
         }
 
+        private void DiscoverSubreports(XDocument doc)
```

Comment:
where is the logic that changes the subreport paths in parent report?

Reply:
if you look at the first commit, in the comments section of "DiscoverSubreport", you can find the renaming logic. i decided to remove it because PBI doesn't care about the subreport path, it somehow parses the path to just the report name and tries to match it from the current workspace. i mentioned this in the PR description.

Reply:
Oh interesting. Yeah we have a logic in backend to ignore the path and only look for file name. Then the only case where it needs modification is same report name in different folders - which is the name conflict case where we want to ignore for now, right?

Reply:
yes, currently we have a reportNameMap dictionary in place to keep track of the names of the reports we will be uploading. for cases where reports have the same name from different folders, the one from the parent report folder has priority, and more shallowly embedded subreports have priority.